app/src/main/res/values/strings.xml

```diff
@@ -249,6 +249,7 @@
     <string name="nothing_to_scan">Nothing to scan.</string>
     <string name="scanned_files">Scanned %1$d of %2$d files.</string>
     <string name="could_not_scan_files">Could not scan %d files.</string>
+    <string name="scanning_x_songs_so_far">Scanning songs - %1$d so far…</string>
```

Comment:
is it possible to display the total number of songs? so we could have "Scanning song x of y"

Reply:
It's not that easy - since the real work of song scanning is done asynchronously, there might be multiple callers that request a song before that one gets scanned and added to the discog cache.

To get the Y number, we need some kind of dedupe on the queued scan requests. probably by the song id

Reply:
but I agree that this would be more meaningful to display "scanning x over y" than the current proposal.

right now the only counter measure to this race condition is this one
https://github.com/soncaokim/VinylMusicPlayer/blob/65d22a1456fa28fa9488b55182cbcaab131b21b9/app/src/main/java/com/poupa/vinylmusicplayer/discog/Discography.java#L136

If we want to dedupe, the async task queue needs to be managed by us (and not by the OS).

Reply:
I see. This was a nice to have, definitely not something that should hold the PR back